pkg/restapi/rp/operation/operations.go

```diff
@@ -80,10 +94,27 @@ func (o *Operation) hydraConsentHandler(w http.ResponseWriter, _ *http.Request)
 	testResponse(w)
 }
 
-// Frontend requests a request for a presentation and provides a handle.
-func (o *Operation) getPresentationRequestHandler(w http.ResponseWriter, _ *http.Request) {
-	// TODO extract handle and return the request for presentation.
-	testResponse(w)
+// Frontend requests to create presentation definition.
+func (o *Operation) createPresentationDefinition(rw http.ResponseWriter, req *http.Request) {
```

Comment:
This endpoint is called by the frontend. The user is redirected to the frontend in the consent phase, and the backend already has the scopes. It doesn't make sense to pass the scopes to the frontend for it to then pass it back to the backend to create the presentation. It also opens up this endpoint to replay exploits, etc. The idea here was to have the backend create a unique handle of one-time use to a presentation, and pass that to the frontend. The frontend would call the backend with this handle and get the presentation.

Reply:
I didn't know about this unique handle

Reply:
@llorllale who is working on unique handle

Reply:
@troyronda @sudeshrshetty @rolsonquadras ^^^^

Reply:
@fqutishat it is in the TODO comment delete in this PR, and also in `hydraConsentHandler`

Reply:
@fqutishat see #14

Reply:
@llorllale I add TODO #14